packages/onboarding/package.json

```diff
@@ -32,6 +32,7 @@
 		"@automattic/calypso-config": "workspace:^",
 		"@automattic/components": "workspace:^",
 		"@automattic/data-stores": "workspace:^",
+		"@automattic/shopping-cart": "workspace:^",
```

Comment:
This package already uses the `shopping-cart` package, but it wasn't listed in the dependencies for some reason.